src/backEnd/controllers/login.js

```diff
@@ -0,0 +1,37 @@
+const bcrypt = require('bcryptjs');
+const jwt = require('jsonwebtoken');
+const { user } = require('../../database/models');
+
+const { SECRET } = process.env;
+
+exports.get = (req, res) => {
+  res.render('login');
+};
+
+exports.post = (req, res) => {
+  const {
+    email, password,
+  } = req.body;
+  user.findOne({
+    where: { email },
+    attributes: ['id', 'email', 'password'],
+    raw: true,
+  }).then((result) => {
+    if (!result) {
+      res.render('login', { message: 'email or password not match' });
+      return;
+    }
+    bcrypt.compare(password, result.password, (err, result2) => {
+      if (err) {
+        res.send(err);
+      }
+      if (!result2) {
```

Comment:
if(err) & if(!result2)  here are same, if there's not an error that's mean the result exists, no result means error's exist 
```js 
if(err) {
res.render('login', { message: 'email or password not match' });
} else {
 const token = jwt.sign({ id: result.id, email }, SECRET);
        res.cookie('logged_in', token, { maxAge: 999999999 }).render('profile');
}
```